projects/hslayers/src/common/layers/public-api.ts

```diff
@@ -1,3 +1,4 @@
 export * from './hs.source.SPOI';
 export * from './hs.source.SparqlJson';
 export * from './hs.source.WfsSource';
+export * from './interpolated-source';
```

Comment:
I am not pushing you to anything, but I would name the file `hs.source.interpolated` just for the sake of consistency in this folder.